protocol/v2/ssv/spectest/sync_committee_aggregator_proof_type.go

```diff
@@ -17,7 +18,7 @@ import (
 func RunSyncCommitteeAggProof(t *testing.T, test *synccommitteeaggregator.SyncCommitteeAggregatorProofSpecTest) {
 	ks := testingutils.Testing4SharesSet()
 	share := testingutils.TestingShare(ks)
-	logger := logging.TestLogger(t)
+	logger := logging.GetLogger(zap.DebugLevel)
 	v := ssvtesting.BaseValidator(logger, keySetForShare(share))
```

Comment:
so you can use like this
```suggestion
	logging.SetGlobalLogger(zap.DebugLevel)
	v := ssvtesting.BaseValidator(zap.L(), keySetForShare(share))
```